code/modules/organs/blood.dm

```diff
@@ -196,14 +196,17 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 
 			blood_max = blood_max * BLOODLOSS_SPEED_MULTIPLIER
```

Comment:
I think this one's also probably not supposed to be per organ

Reply:
You're probably right, thank you ancient shitcoders